packages/api/src/auth/verifiers/sha256Verifier.ts

```diff
@@ -0,0 +1,112 @@
+import { createHmac, timingSafeEqual } from 'crypto'
+
+import {
+  VerifyOptions,
+  WebhookVerificationError,
+  VERIFICATION_ERROR_MESSAGE,
+  DEFAULT_WEBHOOK_SECRET,
+} from './index'
+import type { WebhookVerifier } from './index'
```

Comment:
Looks like a circular dependency here, typescript should probably complain about this.

e.g. When importing this verifier type in index.ts
Sha256Verifier -> WebhookVerifier